edm.yaml

```diff
@@ -192,21 +192,41 @@ datatypes :
      - fcc::CaloHit hits // Hits used in the cluster
 
   fcc::TrackState:
-    Description: "Track state at a given point on the track."
+    Description: "Track state at a given point on the track. The track parameters are chosen 
```

Comment:
This description is not consistent with the parameters of the member section below. For example, one is talking about q/p here, but in the member section, one gets momentum and charge. 
btw, should probably use a Vector3 FCC component instead of an array if one wants to store the momentum.